connect-button/src/main/java/com/ifttt/connect/ui/BaseConnectButton.java

```diff
@@ -1137,6 +1143,17 @@ public void onActivityResumed(Activity activity) {
                     activityLifecycleCallbacks = null;
                 }
             }
+
+            @Override
+            public void onActivityPaused(Activity activity) {
+                /*
+                * Clear the event queue when activity is paused to make sure the queued events are not missed
+                * */
+                if (activity == context) {
```

Comment:
I don't think we need to be this aggressive sending the batched events, especially this callback is only used when the webview is open.

Reply:
We may lose the connect button click event for certain scenarios if we don't flush events here.

Reply:
Talked irl, and we don't need this aggressive queue flush here.